src/content/en/updates/2019/08/twas-quickstart.md

```diff
@@ -0,0 +1,221 @@
+[here](https://developer.android.com/studio/publish/app-signing#generate-key).
+Once you created your key, hit next and pick the “release” build variant.
+Make sure you tick both the V1 and the V2 signatures (the Play Store won’t let you upload the APK
+otherwise).
+Hit Finish.
+
+After a few seconds a pop up will appear in the bottom right corner giving you the option to locate
+or analyze the APK (if you miss it, you can press on the *Event Log* in the bottom right).
+Annoyingly you’ll need to use adb manually to install the signed APK with
+`adb install app-release.apk`.
+
+<table>
+  <tr><th>Key</th><th>Debug</th><th>Release</th></tr>
+  <tr>
+    <td>When is it created?</td>
+    <td>Automatically by Android Studio.</td>
+    <td>Manually by you.</td>
+  </tr>
+  <tr>
+    <td>When is it used?</td>
+    <td><ul>
+      <li><i>Run 'app'</i>.</li>
+      <li><i>Debug 'app'</i>.</li>
+      <li><i>Build APK</i>.</li>
+    </ul></td>
+    <td><ul>
+      <li><i>Generate Signed APK</i>.</li>
+      <li>When the app is downloaded from the Play Store.</li>
+    </ul></td>
+  </tr>
+</table>
+
+## Creating your asset link file {: #creating-your-asset-link-file }
+
+Now that you have your app installed (with either the debug or release key) we can go about
+generating the Digital Asset Link file.
+I’ve created the
+[Asset Link Tool](https://play.google.com/store/apps/details?id=dev.conn.assetlinkstool) to help you
+do this - download that app and run it.
+
+When the app launches, you’ll be given a list of all applications installed on your device by
+`applicationId`.
+Filter the list by the `applicationId` you chose earlier and click on that entry.
+You’ll see a page listing your app’s signature and with a generated Digital Asset Link.
+Click on the Copy or Share buttons at the bottom to export it however you like (eg, save to Google
+Keep, email it to yourself).
+
+Put that JSON in a file called `assetlinks.json` and upload it to your website at
+`.well-known/assetlinks.json` (relative to the root).
+
+### Ensuring your asset link file is accessible {: #ensuring-your-asset-link-file-is-accessible }
+
+Now that you’ve uploaded it, make sure you can access your asset link file on a browser - check that
+`https://yourwebsite.com/.well-known/assetlinks.json` resolves to the file you just uploaded.
+
+Jekyll based websites
+If your website is generated by Jekyll (such as GitHub Pages), you’ll need to add a line of
+configuration so that the `.well-known` directory is included in the output.
+Create a file called `_config.yml` at the root of your site (or add to it if it already exists) and
+enter (more information
+[here](https://help.github.com/en/articles/files-that-start-with-an-underscore-are-missing)):
+
+```yml
+# Folders with dotfiles are ignored by default.
+include: [.well-known]
+```
+
+### Adding more keys {: #adding-more-keys }
+
+A Digital Asset Link file can contain multiple apps that are verified for your website and can allow
```

Comment:
@jpmedley - I think there's an easier way to say this, but I couldn't figure it out. An asset file can contain more than one app, and for each app, it can contain more than one key.